R/mmrm-methods.R

```diff
@@ -48,26 +48,12 @@ h_coef_table <- function(object) {
 #' # Summary:
 #' summary(object)
 summary.mmrm <- function(object, ...) {
-  aic_list <- list(
-    AIC = AIC(object),
-    BIC = BIC(object),
-    logLik = logLik(object),
-    deviance = deviance(object)
-  )
   structure(
-    list(
-      logLik = logLik(object),
-      cov_type = object$formula_parts$corr_type,
-      n_theta = length(object$theta_est),
-      n_subjects = object$tmb_data$n_subjects,
-      n_timepoints = object$tmb_data$n_visits,
-      n_obs = length(object$tmb_data$y_vector),
-      coefficients = h_coef_table(object),
-      vcov = vcov(object),
-      varcor = VarCorr(object),
-      aic_list = aic_list,
-      call = stats::getCall(object)
-    ),
+    Reduce(append,
```

Comment:
hm this makes the code less readable, maybe you could instead just `c()` together the `aic_list` with the `components` list?

Reply:
updated but unclear if it's an improvement.